install.sh

```diff
@@ -45,13 +47,15 @@ EOF
   FORCE_FLAG=
   CLEAN_UP_FLAG=
   PRE_FLIGHT_FLAG=
-  while getopts hfdp?c: name ; do
+  LICENSE_TEST_FLAG=
+  while getopts hfdpl?c: name ; do
       case $name in
       h)    HELP_FLAG=1; show_help;;  # Help
       c)    CONFIG_FILE="${OPTARG}";; # Config file name to install - this overrides the default, 'config.tfvars'
       f)    FORCE_FLAG="-f";;         # Auto-approve
       d)    CLEAN_UP_FLAG="-d";;      # Run cleanup script before install
       p)    PRE_FLIGHT_FLAG="-p";;    # Skip pre-flight checks to test compatibility of EBS and RDS snapshots if any
+      l)    LICENSE_TEST_FLAG="-l";;  # Skip license checks
```

Comment:
```suggestion
      l)    SKIP_LICENSE_TEST_FLAG="-l";;  # Skip license checks
```
I would expect `LICENSE_TEST_FLAG` to actually enable the test.